pallets/valve/src/lib.rs

```diff
@@ -213,7 +213,7 @@ pub mod pallet {
 		pub fn open_pallet_gates(origin: OriginFor<T>) -> DispatchResult {
 			ensure_root(origin)?;
 
-			ClosedPallets::<T>::remove_all(Some(5));
+			let _ = ClosedPallets::<T>::clear(5, None);
```

Comment:
`remove_all` was deprecated. Clear is basically remove_all with a cursor. We pass in `None` at first since we have no cursor, and don't use the response as we don't need the cursor

Reply:
correct